github_issue_tools/create_github_issue.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Load the environment variables from the .env file.
+if [[ -f ".env" ]]; then
+    source .env
+else
+    echo -e "Error: The '.env' file does not exist.\nYou can create one using the '.env.template' file."
+    exit 1
+fi
+
+echo "Starting script"
+
+# Read the CSV file and extract the column values.
+{
+  # Read the header of the CSV file.
+  read
+  while IFS="," read -r title owner repo description labels milestone
+  do
+    # Create API request body.
+    body='{"title":"'"$title"'","body":"'"$description"'"'
+
+    # Add label in the request body, if present.
+    if [ "$labels" != "" ] 
+    then
+      # Extract a single label into multiple labels separated by /.
+      labels=$(echo "$labels" | sed 's/\//","/g') 
+      body=$body',"labels":["'"$labels"'"]'
```

Comment:
Labels can have slashes in them, so I'm thinking this won't work. We can instead use a vertical bar `|` for the separator, as I don't think it will be used anywhere.

Also, I'm thinking using `jq` may make this sort of operation more readable:

https://stackoverflow.com/a/48470227

Not sure if this works:

```bash
labels=$(echo "$labels" | sed 's/|/","/g')

JSON_STRING=$( jq -n \
                  --arg title "$title" \
                  --arg body "$description" \
                  --arg labels "$labels" \
                  '{title: $title, body: $body, labels: [$labels]}' )
```